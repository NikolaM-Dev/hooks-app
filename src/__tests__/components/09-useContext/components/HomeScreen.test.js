import '@testing-library/jest-dom';
import { mount } from 'enzyme';

import HomeScreen from 'components/09-useContext/components/HomeScreen';
import UserContext from 'components/09-useContext/UserContext';

describe('Tests on <HomeScreen />', () => {
  const USER = {
    name: 'Nikola',
    email: 'nikola@gmail.com',
  };

  it('should show <HomeScreen /> correctly', () => {
    const wrapper = mount(
      <UserContext.Provider value={{ user: USER }}>
        <HomeScreen />
      </UserContext.Provider>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
