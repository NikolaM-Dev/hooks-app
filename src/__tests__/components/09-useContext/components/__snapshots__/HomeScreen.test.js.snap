// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <HomeScreen /> should show <HomeScreen /> correctly 1`] = `
Array [
  <h1>
    Home Screen
  </h1>,
  <hr />,
  <pre>
    {
   "name": "Nikola",
   "email": "nikola@gmail.com"
}
  </pre>,
]
`;
